scripts/build_wheel.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I haven't looked at the build scripts very closely - I expect we will test them, and possibly update them, as part of setting up the wheel creation and testing in the workflows.